### ISSUE TITLE
File Input Componnent Accessibility

### ISSUE THREAD

**Issue by bryan-robitaille** [cds-snc/platform-forms-client#379]
As a user I should be able to properly navigate and interact with File Upload fields using assistive technology.

According to various a11y groups the basic file input field is not very accessible:
https://a11ysupport.io/tech/html/input(type-file)_element

The current file upload component needs to be redesigned to:
- be as accessible as possible
  - can provide current file specified to upload
  - can provide feedback that the current value has changed
- ensure the filename is still shown when the form re-renders after a validation check

**Comment by bryan-robitaille** [comment]
@falila @craigzour Hey team! Can you please add your [planning poker estimates with ZenHub](https://app.zenhub.com/workspaces/GCForms-60cb8929764d71000e481cab/issues/cds-snc/platform-forms-client/379?planning-poker)

**Comment by Moro-Code** [comment]
Think this is done

**Comment by bryan-robitaille** [comment]
Closed by PR https://github.com/cds-snc/platform-forms-client/pull/385
